web/src/main/java/com/google/sps/servlets/SaveStockServlet.java

```diff
@@ -37,26 +39,33 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Sanitize user input to remove HTML tags and JavaScript.
 
     Document doc = Jsoup.connect("https://coinmarketcap.com/").get();
-    String websitedata = doc.html(); // prints HTML data
+    String websiteData = doc.html(); // prints HTML data
     System.out.println("✔️GOT HTML PAGE");
 
     Elements tik = doc.select(".coin-item-symbol");
     Elements price = doc.select(".price___3rj7O ");
     System.out.println("✔️GOT ALL TICKER SYMBOLS IN PAGE");
     System.out.println("✔️GOT ALL PRICES IN PAGE");
 
-    long timestamp = System.currentTimeMillis();
+    long timeStamp = System.currentTimeMillis();
 
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
     KeyFactory keyFactory = datastore.newKeyFactory().setKind("Stock");
+
     for (int i = 0; i < tik.size(); i++) {
+        
       String tikk = tik.get(i).text();
       String pricee = price.get(i).text();
+
+      String strippedPrice = pricee.replaceAll("[\\\\$,]", "");
+      double priceDouble = Double.parseDouble(strippedPrice);
+
       FullEntity taskEntity =
           Entity.newBuilder(keyFactory.newKey())
               .set("Tik", tikk)
-              .set("Price", pricee)
-              .set("timestamp", timestamp)
+              .set("DisplayPrice", pricee)
+              .set("TimeStamp", timeStamp)
+              .set("DoublePrice", priceDouble)
```

Comment:
Let's change "DoublePrice" to "USD" since we want to make sure we know what currency it is representing.